content/api/vision.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Vision
+---
+
+{{% notice info %}}
```

Comment:
I think you removed the wrong thing. I meant remove this notice, not the opening sentence 😂

Reply:
Well heck!